src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -89,12 +66,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String[] uriList = request.getRequestURI().split("/");
     assert (uriList[1].equals("user"));
 
+    if (uriList.length < 3) {
+      response.sendError(
+          HttpServletResponse.SC_METHOD_NOT_ALLOWED, "Unimplemented: " + request.getRequestURI());
+      return;
+    }
+
+    // Replace 'current' user with logged-in user.
+    String userKey = uriList[2];
+    if (userKey.equals("current")) {
```

Comment:
Can you go through why this value would be `"current"` for users who are not logged in? Also would be nice to have this as a doc-level const just like `CURRENT_USER_KEY`.

Reply:
When the server requests `/user/current`, instead of obtaining the relevant user key from the URL, it obtains the user key of whoever is logged in. If the user with key `1` is logged in, `/user/current` is equivalent to `/user/1`.

I can definitely change `"current"` to a doc constant though.

Reply:
I can also convert similar string literals to constants.